File: Java/Tests/jdi-uitest-tutorialtests/src/main/java/com/epam/jdi/uitests/testing/career/page_objects/entities/JobSearchFilter.java
package com.epam.jdi.uitests.testing.career.page_objects.entities;

import com.epam.jdi.uitests.testing.career.page_objects.enums.JobCategories;

import static com.epam.jdi.uitests.testing.career.page_objects.enums.JobCategories.QA;
import static com.epam.jdi.uitests.testing.career.page_objects.enums.Locations.SAINT_PETERSBURG;

/**
 * Created by Roman_Iovlev on 10/22/2015.
 */
public class JobSearchFilter {
    public CharSequence keywords = "QA";
    public JobCategories category = QA;
    public String location = SAINT_PETERSBURG.value;
}

File: C#.Net/Tests/JDI Tests/Entities/JobSearchFilter.cs
using JDI_Tests.Enums;
using OpenQA.Selenium;

namespace JDI_Tests.Entities
{
    public class JobSearchFilter
    {
        public string Keywords = "Test" + Keys.Return;
        public string Category = "Software Test Engineering";
        public string Location = Locations.SaintPetersburg;
    }
}
